src/rmarkdown/preview.ts

```diff
@@ -38,26 +38,26 @@ class RMarkdownPreview extends vscode.Disposable {
         this.resourceViewColumn = resourceViewColumn;
         this.outputUri = outputUri;
         this.autoRefresh = autoRefresh;
-        void this.refreshContent(themeBool);
+        void this.refreshContent(useDarkTheme);
```

Comment:
Not sure whether it works if we use a high-contrast theme.

Reply:
Maybe something like useCodeTheme?